Dockerfile

```diff
@@ -21,13 +21,14 @@ RUN \
 	go version
 
 # install goreleaser
-ARG GORELEASER_VERSION=1.1.0
-ARG GORELEASER_SHA=10a6356fc1762458b4e4bbb388d0daab182f2eca2c314b8790b8017ca1e284d1
+ARG GORELEASER_VERSION=1.2.2
+ARG GORELEASER_SHA=4ef68da9014df222c94ccefa02b01fdf96d414c1c775dbee5fadda53cd74ff71
 RUN  \
 	GORELEASER_DOWNLOAD_FILE=goreleaser_Linux_x86_64.tar.gz && \
 	GORELEASER_DOWNLOAD_URL=https://github.com/goreleaser/goreleaser/releases/download/v${GORELEASER_VERSION}/${GORELEASER_DOWNLOAD_FILE} && \
-	cosign verify-blob --key https://github.com/goreleaser/goreleaser/releases/download/v$GORELEASER_VERSION/cosign.pub \
+	cosign verify-blob \
 	--signature https://github.com/goreleaser/goreleaser/releases/download/v$GORELEASER_VERSION/checksums.txt.sig \
+    --cert https://github.com/goreleaser/goreleaser/releases/download/v$GORELEASER_VERSION/checksums.txt.pem \
```

Comment:
I think cert does not accept an URL

Reply:
so, let's add this support to cosign 🤩

Reply:
Works for me!

Reply:
https://github.com/sigstore/cosign/pull/1245 :D

Reply:
@cpanato let's wait for that before merging this, WDYT?

Reply:
sgtm

Reply:
y'all are so fast 😂